scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -356,7 +356,7 @@ <h6 class="mt-3 ms-3">
                   <a class="text-dark" href="{{ url_for('cases.case', institute_id=grouped_case.owner, case_name=grouped_case.display_name) }}">{{ grouped_case.display_name }}</a>
 
                   <a href="{{ url_for('cases.remove_case_group', institute_id=institute._id, case_name=grouped_case.display_name, case_group=group_id) }}" class="btn btn-link btn-sm">
-                      <span class="fa fa-remove"></span></a>
+                      <span class="fa fa-remove text-black"></span></a>
```

Comment:
i'm not really satisfied by this. Perhaps just remove the btn-link from the line above?

Reply:
Also sounds like a plan!